examples/index.json

```diff
@@ -1072,7 +1072,7 @@
     {
         "title": "Lock",
         "column": 0,
```

Comment:
I think this should go to column 2 after the Concurrency examples maybe?

Reply:
will fix